docs/modules/servers/partials/configure/blobstore.adoc

```diff
@@ -0,0 +1,248 @@
+== BlobStore
+
+This file is optional. If omitted, the *cassandra* blob store will be used.
+
+BlobStore is the dedicated component to store blobs, non-indexable content.
+James uses the BlobStore for storing blobs which are usually mail contents, attachments, deleted mails...
+
+You can choose the underlying implementation of BlobStore to fit with your James setup.
+
+It could be the implementation on top of Cassandra or file storage service S3 compatible like Openstack Swift and AWS S3.
+
+Consult link:{sample-configuration-prefix-url}/blob.properties[blob.properties]
+in GIT to get some examples and hints.
+
+=== Implementation choice
+
+*implementation* :
+
+* cassandra: use cassandra based BlobStore
+* objectstorage: use Swift/AWS S3 based BlobStore
+* file: (experimental) use directly the file system. Useful for legacy architecture based on shared ISCI SANs and/or
+ distributed file system with no object store available.
+* postgres: use Postgres based BlobStore
```

Comment:
Idem... You could have for example:

* {dbName}: use {dbName} based BlobStore

Instead of giving both items of cassandra and postgresql in the partial. You can't have cassandra implem in postgres app and can't have postgres implem in distributed app :)

Reply:
It's not the only place, it's not just simply replacing the word 'cassandra' with 'postgres'. 
To be clearer, I will move the entire section: 
=== Implementation choice 
=== Cassandra BlobStore Cache

out of the partial.

Reply:
Out of the partial is fine, but yeah in the partial was not :)